src/screens/HomeScreen.tsx

```diff
@@ -148,7 +146,7 @@ const HomeScreen = (props: Props) => {
             }}>
             <Text style={styles.hiddenE2e}>End to End Tests</Text>
           </TouchableOpacity>
-          {currentDSN === SENTRY_INTERNAL_DSN && (
```

Comment:
this was popping up every time, even when I had a DSN, so changed logic so it appears if you _don't_ have one set.